R/RcppExports.R

```diff
@@ -251,3 +281,17 @@ swe_topo_arcus_visionis <- function(tjd_ut, dgeo, datm, dobs, helflag, mag, AziO
     .Call(`_swephR_topo_arcus_visionis`, tjd_ut, dgeo, datm, dobs, helflag, mag, AziO, AltO, AziS, AziM, AltM)
 }
 
+#' Compute heliacal event details
+#' @param mag   The object's magnitude
+#' @param AziO  The object's azimut
+#' @param AziS  The sun's azimut
+#' @param AziM  The moon's azimut
+#' @param AltM  The moon's altitude
```

Comment:
You should not repeat these five argument descriptions here, since they have been mentioned with `swe_topo_arcus_visionis`. Since currently all documentation goes onto one page, one has to be careful to not repeat any arguments ...